macros/query_tags.sql

```diff
@@ -63,13 +67,3 @@
     {% do run_query("alter session set query_tag = '{}'".format(new_query_tag)) %}
     {{ return(original_query_tag)}}
 {% endmacro %}
-
-{% macro unset_query_tag(original_query_tag) -%}
```

Comment:
Unless I'm mistaken, this needs to be overridden because the [default](https://github.com/dbt-labs/dbt-snowflake/blob/main/dbt/include/snowflake/macros/adapters.sql#L227) only unsets if a query tag config is set. I added this to make sure only relevant queries are tagged.

Reply:
Fair point. Something we've overlooked in our implementation. We do though have a default tag set in the project so it's never surfaced as an issue.